src/Symfony/Component/Scheduler/Trigger/DatePeriodTrigger.php

```diff
@@ -21,6 +21,16 @@ public function __construct(
     ) {
     }
 
+    public function __toString(): string
+    {
+        return \sprintf(
+            '%s - %s, %s',
+            $this->period->getStartDate()->format(\DateTimeInterface::ATOM),
+            $this->period->getEndDate()?->format(\DateTimeInterface::ATOM) ?? '(inf)',
+            'every ?',
```

Comment:
Not sure what to put here but if #49817 is merged, this won't be necessary.

Reply:
Let's use `-` for now to allow this PR to be merged quickly (#49817 will be merged before 6.3 final, we just need to figure out what we want to do there).